GameServerLib/Game.cs

```diff
@@ -232,19 +238,62 @@ public void InitializePacketHandlers()
             RequestHandler.Register<ViewRequest>(new HandleView(this).HandlePacket);
         }
 
+        /// <summary>
+        /// Enables or disables the hot reloading of scripts. Used only for development.
+        /// </summary>
+        public void EnableHotReload(bool status)
+        {
+            string scriptsPath = Config.ContentManager.ContentPath;
+
+            void ScriptsChanged(object _, FileSystemEventArgs ea)
+            {
+                // Disable raising events to avoid triggering LoadScripts() many times in a row after the first event
+                ScriptsHotReloadWatcher.EnableRaisingEvents = false;
+                ChatCommandManager.SendDebugMsgFormatted(DebugMsgType.INFO, LoadScripts() ? "Scripts reloaded." : "Scripts failed to reload.");
+                ScriptsHotReloadWatcher.EnableRaisingEvents = true;
+            }
+
+            if (status && ScriptsHotReloadWatcher == null)
+            {
+                ScriptsHotReloadWatcher = new FileSystemWatcher
+                {
+                    Path = scriptsPath,
+                    IncludeSubdirectories = true,
+                    EnableRaisingEvents = true,
+                    NotifyFilter = NotifyFilters.LastWrite,
+                    Filter = "*.*",
```

Comment:
The filter should be `"*.cs*"`.

Reply:
Does not work for some odd reason.

Reply:
Try `"*.cs"` instead.

Reply:
It's very strange, but it only seems to work with "*.*"

Reply:
It works fine with `"*.cs"`, changing the file via Visual Studio while debugging will not trigger the file changed event. However, changing the file via a text editor will.

Reply:
Do you think we should keep it so that the files can be edited with Visual Studio? The watcher will only run during development when explicitly enabling hot reload anyways

Reply:
It isn't entirely necessary I agree, however I believe it should at least be kept as an issue for the future. You should be able to merge as-is, and I'll just make an issue.

Reply:
I agree.